spyder/plugins/ipythonconsole/widgets/help.py

```diff
@@ -38,14 +38,20 @@ def clean_invalid_var_chars(self, var):
         return re.sub(r'\W|^(?=\d)', '_', var)
 
     def get_documentation(self, content):
-        """Get documentation fron inspect reply content."""
+        """Get documentation from inspect reply content."""
         data = content.get('data', {})
         text = data.get('text/plain', '')
-        signature = self.get_signature(content).split('(')[-1]
         if text:
-            documentation = text.split(signature)
-            if len(documentation) > 1:
-                return documentation[-1]
+            text = re.compile(ANSI_PATTERN).sub('', text)
```

Comment:
Why is this necessary?

Reply:
This is needed to remove characters like `[0m` but no characters like `new line` (`\n`):

![without_p](https://user-images.githubusercontent.com/16781833/59568144-63e68500-903c-11e9-800c-b8d1e5264f8f.png)

Reply:
Ok, thanks for the explanation.